src/lib/grammar.rs

```diff
@@ -386,4 +389,29 @@ mod test {
         assert_eq!(grm.prod_precs[6].unwrap(), Precedence{level: 4, kind: AssocKind::Nonassoc});
         assert!(grm.prod_precs[7].is_none());
     }
+
+    #[test]
+    fn test_prec_override() {
+        let grm = Grammar::new(&parse_yacc(&"
+            %start expr
+            %left '+' '-'
+            %left '*' '/'
+            %%
+            expr : expr '+' expr
+                 | expr '-' expr
+                 | expr '*' expr
+                 | expr '/' expr
+                 | %prec '-'  expr '*'
```

Comment:
That's the example Yacc uses but, as I read it, it's pointless because each production gets the precedence of the last terminal in the production by default? So `'-' expr '*'` has the same precedence as `'-' expr %prec '*'` I think? Warning: I might be wrong